### ISSUE TITLE
The download button on isis cardview doesn't have enough contrast

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#899]
**Description**:

The download button doesn't have sufficient contrast.

![download(dark mode)](https://user-images.githubusercontent.com/83226114/139670527-c753bb60-b524-4232-9f8f-92991b3b27c1.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done

**Comment by joelvdavies** [comment]
@joshuadkitenge It seems isisDatasets/InvestigationsCardView uses an outlined variant of the download button and nowhere else so maybe it is best to change these?
